.docker/aiida-core-dev/Dockerfile

```
@@ -0,0 +1,9 @@
+# syntax=docker/dockerfile:1
+FROM aiida-core-with-services
+
+LABEL maintainer="AiiDA Team <developers@aiida.net>"
+
+COPY aiida-clone-and-install.sh /etc/init/run-before-daemon-start/
```

```suggestion
COPY aiida-clone-and-install.sh /etc/init/run-before-daemon-start/10-aiida-clone-and-install.sh
```

The scripts in the folder is run by linux `run-parts` which can run in order by depending on the number in front.

---

Nice 👍